git-multimail/git_multimail.py

```diff
@@ -2171,7 +2174,7 @@ def call(klass, server, timeout):
                 % (self.smtpserver, sys.exc_info()[1]))
             sys.exit(1)
 
-    def __del__(self):
+    def close(self):
```

Comment:
I think it's better to also keep a `__del__` method, so that any user of the class get the proper connection even without calling `close`. Not strictly needed, but it's just a 2-liners.